### PR TITLE
Added nested dialogue to DialogView allowing quickly adding relations

### PR DESCRIPTION
**Pull request by thealmightyone**

### CODE REVIEW

File: app/js/views/dialogView.js [hunk 1]
```diff
@@ -90,17 +93,100 @@ export default class DialogView extends View {
...
+    };
+    const onhide = () => {
+      const editor = this.form.fields[passedSchema.id + '_id'].editor;
+
+      if (editor.getValue() === 'addNew' + passedSchema.id) {
+        editor.setValue(undefined);
+      }
+    };
+
+    passedSchema.filterByAction(action, this.parentProperty).then(schema => {
```
Comment (stepek) [inline app/js/views/dialogView.js]: Please add callback for reject, in callback you can put console.error message.

File: app/js/views/dialogView.js [hunk 2]
```diff
@@ -90,17 +93,100 @@ export default class DialogView extends View {
...
+      }
+    }
+  }
   /**
    * Renders dialog view.
    * @returns {DialogView}
    */
   render() {
+    this.updateDialogSchema();
+    Backbone.Form.editors.List.Object = Backbone.Form.editors.Object;
```
Comment (stepek) [inline app/js/views/dialogView.js]: please check this line https://github.com/cloudwan/gohan_webui/commit/87e80f9f8263d7dbe2d766d4c474b0ff5eea7704#diff-3ae4d23a0107af917e74d064adff92dbL94 
I removed this line.

**Comment by stepek** [comment]
LGFM

**Comment by stepek** [comment]
@thayashi can you check this?

**Comment by thayashi** [comment]
Cool, thanks. As implementation, no problem, I'd like to have option to disable this feature.  In some case, it might be better to not to show this feature. May be in config.json?

**Comment by stepek** [comment]
Thanks @thayashi.
@thealmightyone Please add option to disable this feature.

**Comment by stepek** [comment]
@thealmightyone Please rebase this commit.

File: app/config.json [hunk 3]
```diff
@@ -0,0 +1,17 @@
+{
```
Comment (stepek) [inline app/config.json]: Please remove this file from this commit.

File: app/js/views/dialogView.js (hunk 1 above)
Comment (stepek) [inline app/js/views/dialogView.js]: done

**Comment by stepek** [comment]
LGFM

**Pull request merged**